next/components/forms/widgets/ExampleInput.tsx

```diff
@@ -10,6 +13,15 @@ export const ExampleInput = (props: any) => {
         required={props.required}
         onChange={(event) => props.onChange(event.target.value)}
       />
+      <Button variant='link-black' href='/stackoverflow.com/' label='Link value'></Button>
+      {/* <Button variant='link-brand' size='sm' href='https://stackoverflow.com/' label='Link value'></Button> */}
+      <Button size="sm" icon={<SearchIcon />} variant="brand-outline"></Button>
+      <Button text="Button" variant="black" size="sm"></Button>
+      <Button icon={<SearchIcon />} variant="black-outline"></Button>
+      <Button text="Button" variant="negative"></Button>
+      <Button text="Button" variant="plain-brand"></Button>
+      <Button text="Button" variant="plain-black" size="sm"></Button>
+      <Button text="Button" variant="plain-negative"></Button>
```

Comment:
I'm all for building a 'showcase' page where you can see all the variants next to each other, but ExampleInput.tsx is not a right place for it

ok to keep as is in this PR, but we should move it afterwards - will create a subsequent task, the goal will be to have something like this: https://marianum-next.staging.bratislava.sk/styleguide